blazor/dropdown-list/style.md

```diff
@@ -123,4 +145,115 @@ Use the following CSS to customize the appearance of popup element.
     font-family: emoji;
     min-height: 29px;
 }
-```
\ No newline at end of file
+```
+
+## Change the HTML attributes
+
+You can add the additional html attributes such as styles, class, and more to the root element using [HTMLAttributes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_HtmlAttributes) property
```

Comment:
Add the additional html attributes such as styles, class, and more to the root element using the [HTMLAttributes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_HtmlAttributes) property.